sqlglot/dialects/clickhouse.py

```diff
@@ -463,48 +463,51 @@ def _parse_function(
             optional_parens: bool = True,
             any_token: bool = False,
         ) -> t.Optional[exp.Expression]:
-            func = super()._parse_function(
+            expr = super()._parse_function(
                 functions=functions,
                 anonymous=anonymous,
                 optional_parens=optional_parens,
                 any_token=any_token,
             )
 
+            func = expr.this if isinstance(expr, exp.Window) else expr
+
             if isinstance(func, exp.Anonymous):
                 parts = self.AGG_FUNC_MAPPING.get(func.this)
+                is_combined = parts and parts[1]
```

Comment:
This seems hacky, we shouldn't be checking `parts[1]` as a proxy for >= 2 items. Let's be explicit by using `len`.

Reply:
Nvm ignore me, I forgot `parts` is a pair.